pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -2781,7 +2797,11 @@ void GSRendererHW::Draw()
 	// Temporary source *must* be invalidated before normal, because otherwise it'll be double freed.
 	g_texture_cache->InvalidateTemporarySource();
 
-	//
+	// Set the RT format back to 32bits after the shuffle.
+	if (rt && m_texture_shuffle && rt->m_32_bits_fmt && GSLocalMemory::m_psm[rt->m_TEX0.PSM].bpp == 16)
```

Comment:
Shouldn't the RT not be having the format changed in the first place?

Unless it's a new target, I guess. In which case, we could gate this with `TEX0.TBP0 != FRAME.Block()`.

Reply:
Uhhh, I can't remember the scenario, but this has always been a problem xD I think it was a new target, but this has been safe, so far.

But I can gate it, if you want.   I'll run a dump with some logging to check the TBP & FBP first to see what games are hitting it.